src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -18,7 +21,11 @@ export default function SearchBar({ onSearch, userInput, onInputChange }) {
         onSubmit={isGlobalSearch ? null : onSubmit}
         id="resources-support-search"
         method="get"
-        action={isGlobalSearch ? '/search' : '/resources/search'}
+        action={
+          isGlobalSearch
+            ? `${searchSettings.rootUrl}/`
+            : `${resourcesSettings.rootUrl}/`
+        }
```

Comment:
@kelsonic made that change fysa - look good?

Reply:
Oh wow! Nice! I'm going to add this to my PR too, thanks!